backend/src/main/resources/db/init/tables/001_users.sql

```diff
@@ -0,0 +1,18 @@
+--liquibase formatted sql
+
+--changeset qwarty:init/tables/users failOnError:true logicalFilePath:init/tables/users
+
+CREATE TABLE users(
+    id UUID NOT NULL DEFAULT gen_random_uuid(),
```

Comment:
```suggestion
    id UUID PRIMARY KEY NOT NULL DEFAULT gen_random_uuid() ,
```

nit: Since our primary key only consists of one column, maybe we can just do this instead of the more verbose `CONSTRAINT` line below

Reply:
yep, I want to keep it as-is to have flexibility on naming. I'll add it in a README in the `/db` folder